iconservice/utils/msgpack_for_db.py

```diff
@@ -94,4 +94,4 @@ def dumps(cls, data: Any) -> bytes:
 
     @classmethod
     def loads(cls, data: bytes) -> list:
-        return msgpack_loads(data, ext_hook=cls._decode, raw=False)
+        return msgpack_loads(data, ext_hook=cls._decode, raw=False, strict_map_key=False)
```

Comment:
Why don't you use strict_map_key=True instead?

Reply:
`strict_map_key` option disallow `Address` key but `PRepAddressConverter` use `Address` key